lib/digitalocean-deploy-plugin.js

```diff
@@ -73,27 +75,18 @@ module.exports = DeployPlugin.extend({
       distDir: distDir
     });
 
+    var installDependenciesCommand = 'sudo apt-get update -y; sudo apt-get upgrade -y; sudo apt-get install -y nginx gcc build-essential; rm /etc/nginx/sites-enabled/default;';
```

Comment:
Maybe instead of all in one string, can break the commands out and join with the separator?

```diff
- var installDependenciesCommand = 'sudo apt-get update -y; sudo apt-get upgrade -y; sudo apt-get install -y nginx gcc build-essential; rm /etc/nginx/sites-enabled/default;'
+ var installDependenciesCommand = [
+   'sudo apt-get update -y',
+   'sudo apt-get upgrade -y',
+   'sudo apt-get install -y nginx gcc build-essential',
+   'rm /etc/nginx/sites-enabled/default',
+ ].join(';'); 
```